src/App.js

```diff
@@ -1,13 +1,79 @@
-import React from 'react';
+import React from "react";
+import Form from "./components/TodoComponents/TodoForm";
+import List from "./components/TodoComponents/TodoList";
+import "./components/TodoComponents/Todo.css";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todoList: [
+        {
+          task: "Organize Garage",
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: "Bake Cookies",
+          id: 1528817084358,
+          completed: false
+        }
+      ],
+      newTodo: ""
+    };
+  }
+
+  changeHandler = event => {
+    console.log(event.target.value);
+    this.setState({ newTodo: event.target.value });
+  };
+
+  toggleTodo = index => {
+    this.setState({
+      todoList: this.state.todoList.map((item, idx) => {
+        if (index !== idx) {
+          return item;
+        } else {
+          return {
+            ...item,
+            completed: !item.completed
+          };
+        }
+      })
+    });
+  };
+
+  addNewTodo = event => {
+    event.preventDefault();
+    if (!this.state.newTodo) return;
+    this.setState({
+      todoList: [...this.state.todoList, { task: this.state.newTodo, id: Date.now(), completed: false }],
+      newTodo: ""
+    });
+  };
+  clearCompleted = event => {
+    event.preventDefault();
+    this.setState({
+      todoList: this.state.todoList.filter(task => {
+        return task.completed === true;
```

Comment:
This isn't working properly. It's clearing all the task that are not completed, instead of clearing the completed task as was requested. Flip the logic.

Reply:
I fixed that big, just had to change true to false.